src/main/resources/assets/blocky-bubbles/lang/ru_ru.json

```
@@ -0,0 +1,13 @@
+{
+  "blocky-bubbles.options.culling_awareness.enum.bubble_column": "Без изменений (Java Edition)",
+  "blocky-bubbles.options.culling_awareness.enum.non_air": "На воздухе (non-air)",
+  "blocky-bubbles.options.culling_awareness.enum.none": "Везде (no culling)",
+  "blocky-bubbles.options.culling_awareness.name": "Режим отбраковки",
+  "blocky-bubbles.options.culling_awareness.tooltip": "Какие блоки должны вызывать отбраковку соседних граней колонны пузырей.",
+  "blocky-bubbles.options.enable_animations.name": "Включить анимацию",
+  "blocky-bubbles.options.enable_animations.tooltip": "Следует ли анимировать текстуры колонны пузырей (требует перезагрузки ресурсов).",
+  "blocky-bubbles.options.quality.name": "Колонны Пузырей",
```

А второе слово почему с большой? Это же не имя :)
```suggestion
  "blocky-bubbles.options.quality.name": "Колонны пузырей",
```